webapp/src/projects.tsx

```diff
@@ -747,6 +748,13 @@ export class ProjectsCarousel extends data.Component<ProjectsCarouselProps, Proj
                         <span className="header">{lf("New Project")}</span>
                     </div>
                 </div> : undefined}
+                {showCloudProjectsCard ? <div role="button" className="ui card link buttoncard cloudprojectscard" title={lf("Sign in to see cloud projects")}
```

Comment:
super minor suggestion: using `{showCloudProjectCard && ...}` would make this statement slightly more compact.